webapp/app.py

```diff
@@ -215,6 +216,39 @@ def start_career():
     return flask.render_template("/careers/start.html")
 
 
+@app.route("/careers", methods=["GET"])
```

Comment:
(Just to let you know, it's not actually necessary to add `methods=["GET"]`, since that's the default. You can just not provide a second argument, which is what we usually do I think. But I don't mind it at all 'cos maybe there's also value in explicitly telling the developer which HTTP methods are matched. So it's up to you.)